torch/quantization/_correct_bias.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function, division, absolute_import
+import torch
+import torch.nn as nn
+import torch.nn.quantized as nnq
+
+import torch.quantization
+import torch.quantization._numeric_suite as ns
+
+_supported_modules = {nn.Linear, nn.Conv2d}
+_supported_modules_quantized = {nnq.Linear, nnq.Conv2d}
+
+def get_module(model, name):
+    ''' Given name of submodule, this function grabs the submodule from given model
+    '''
+    current = model
+    name = name.split('.')
+    for subname in name:
+        if subname == '':
+            return current
+        current = current._modules[subname]
+    return current
+
+def get_parent_module(model, name):
+    ''' Given name of submodule, this function grabs the parent of the submodule, from given model
+    '''
+    parent_name = name.rsplit('.', 1)[0]
+    if parent_name == name:
+        parent_name = ''
+    return get_module(model, parent_name)
+
+def get_param(module, attr):
+    ''' Sometimes the weights/bias attribute gives you the raw tensor, but sometimes
+    gives a function that will give you the raw tensor, this function takes care of that logic
+    '''
+    param = getattr(module, attr, None)
+    if isinstance(param, nn.Parameter) or isinstance(param, torch.Tensor):
+        return param
+    elif callable(param):
+        return param()
+    return None
+
+class MeanShadowLogger(ns.Logger):
+    r"""A logger for a Shadow module whose purpose is to record the rolling mean
+    of the data passed to the floating point and quantized models
+    """
+    def __init__(self):
+        super(MeanShadowLogger, self).__init__()
+        self.stats["float"] = None
+        self.stats["quantized"] = None
+        self.count = 0
+        self.float_sum = None
+        self.quant_sum = None
+
+    def forward(self, x, y):
```

Comment:
looks like x must be quantized and y must be float, please specify this in the comment/docstring